cli/server_create.go

```diff
@@ -67,8 +69,11 @@ func runServerCreate(cli *CLI, cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if err := cli.ActionProgress(cli.Context, result.Action); err != nil {
-		return err
+	disableProgressBar, _ := cmd.Flags().GetBool("disable-progress")
+	if disableProgressBar == false {
```

Comment:
You don’t only disable progress reporting; you disable action error handling altogether.